composer.json

```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">=7.1",
-        "pimple/pimple": "^1.0",
+        "spryker/container": "^1.0",
```

Comment:
Shouldn't this be `1.0.0`?

Reply:
It must be `^1.1.0` as in the Pimple mock is only available with the next release of the `spryker/container` module.